project/Dependencies.scala

```diff
@@ -5,7 +5,7 @@ import scala.util.Try
 
 object Dependencies {
   val scala211 = "2.11.12"
-  val scala212 = "2.12.13"
+  val scala212 = "2.12.14"
```

Comment:
you'll need something like https://github.com/scalacenter/scalafix/pull/1408/files#diff-cf70a614681abb9e33115a49e0b4a52865bdfcdb24c4c0df4af6546d92f88f5fR163 for MiMa

Reply:
and probably also https://github.com/scalacenter/scalafix/pull/1408/files#diff-8fe42aaf3e5d1a3f8804efa64a6206f5af6a826dea8f6ce51ad0ce8105188858R2 since `ExplicitResultTypes` will complain that `2.12.14` is not supported....

Reply:
Scalafix runs on 2.13, so we should be good!